katas/content/teleportation/index.md

```diff
@@ -0,0 +1,33 @@
+# Teleportation
+
+@[section]({
+    "id": "teleportation__overview",
+    "title": "Overview"
+})
+
+Teleportation quantum kata is a series of exercises designed to get you familiar with programming in Q#. It covers the quantum teleportation protocol which allows you to communicate a quantum state using only classical communication and previously shared quantum entanglement.
```

Comment:
Let's jump right into the protocol description:
```suggestion
Quantum teleportation protocol allows us to communicate a quantum state using only classical communication and previously shared quantum entanglement.
```

Reply:
Changed